frontend/src/pages/ConnectedPages/Game/index.tsx

```diff
@@ -17,12 +17,24 @@ const Game: React.FC<RouteComponentProps> = ({ location }) => {
   const [previewUrl, setPreviewUrl] = useState<string>("");
   const [answers, setAnswers] = useState<Answer[]>([]);
   const [score] = useState<number>(0);
+  const player = useRef<null | HTMLAudioElement>(null);
 
-  const onQuestionUpdateReceived = (question: QuestionUpdateMessage): void => {
-    console.log(`Received question ${question}`);
-    setPreviewUrl(question.previewUrl);
-    setAnswers(question.answers);
+  useEffect(() => {
+    (async function reloadPlayer() {
```

Comment:
I do not like this being in the useEffect, just put a function outside on the bottom of the useEffect